### PR TITLE
translator: add accesslog

### PR DESCRIPTION
**Pull request by zirain**
fixes: #684

output as follow:
```
[2022-11-07T01:44:50.341Z] "GET /get HTTP/1.1" 200 - 0 426 0 0 "-" "curl/7.58.0" "f5dec879-4c62-4a7d-a555-d7c044c9d5f0" "www.example.com" "10.96.77.203:3000"
```

this's not for https://github.com/envoyproxy/gateway/issues/699

### CODE REVIEW

File: internal/xds/translator/accesslog.go [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+)
+
+var (
+	stdoutFileAccessLog = &fileaccesslog.FileAccessLog{
+		Path: "/dev/stdout",
+	}
+
+	listenerAccessLogFilter = &accesslog.AccessLogFilter{
+		FilterSpecifier: &accesslog.AccessLogFilter_ResponseFlagFilter{
+			ResponseFlagFilter: &accesslog.ResponseFlagFilter{Flags: []string{"NR"}},
```
Comment (arkodg) [inline internal/xds/translator/accesslog.go:20]: can you leave a comment here, mentioning why the filter is added here such as `for the case when a route doesnt exist to upstream`

Comment (zirain) [reply internal/xds/translator/accesslog.go:21]: CI fail again

Comment (arkodg) [reply internal/xds/translator/accesslog.go:21]: will rerun it for now, and make #705 is prioritized

**Review by arkodg — APPROVED** [review]
looks great, thanks again @zirain !

**Pull request merged**